src/tool/mod.rs

```diff
@@ -64,9 +68,27 @@ impl Tools {
             mkfat: Tool::find("mkfs.fat", dryrun).map_err(|_| {
                 anyhow!("mkfs.fat is required for creating FAT filesystems. Please install the 'dosfstools' package.")
             })?,
+            // TODO: Technically don't need ext4 if only using btrfs
```

Comment:
need to validate that one of brtfs/ext4 is used, so you can use the `if..else...error` on the mkbrtfs tool

Reply:
Done, but we'll need to improve it if we add more filesystems, same as for the matching code. But tbh the only other ones would be XFS or ZFS.